ksql-streams/src/main/java/io/confluent/ksql/execution/streams/materialization/ks/KsMaterializationFactory.java

```diff
@@ -127,7 +129,8 @@ KsLocator create(
         String stateStoreName,
         KafkaStreams kafkaStreams,
```

Comment:
too bad, we can't get the applicationId from this kafkaStreams instance. do you think we should push for that?

Reply:
That would be nice.  It makes sense for a KafkaStreams instance to have.